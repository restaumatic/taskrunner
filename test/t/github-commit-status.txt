# check output github
# no toplevel
# github keys

export TASKRUNNER_ENABLE_COMMIT_STATUS=1

git init -q
git commit --allow-empty -q -m "Initial commit"

taskrunner -n mytask bash -e -c '
  snapshot -n --commit-status
  echo Success
'
