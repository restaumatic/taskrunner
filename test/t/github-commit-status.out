-- output:
[mytask] stdout | Success
-- github:
Requested access token for installation 123
Updated commit status for fakeowner/fakerepo to {"context":"mytask","description":"not cached","state":"pending","target_url":null}
Requested access token for installation 123
Updated commit status for fakeowner/fakerepo to {"context":"mytask","description":null,"state":"success","target_url":null}
